src/pages/components/feature-card-block.mdx

```diff
@@ -22,16 +22,39 @@ import ComponentFeedback from 'components/ComponentFeedback';
 ## Feature card block &mdash; medium
 Feature card block &mdash; medium is used to present a group of information with a medium size card with an image.
 
+### Content guidance
+
+| Field | Field type | Required   | Cardinality | Max character size (English / translated) |
+| -------- | --------- | ---------- | ---------- | ----------|
+| Headline | Short copy | Yes   | 1 | XS (65 / 85)|
+| [Featured card](https://www.ibm.com/standards/web/carbon-for-ibm-dotcom/components/feature-card) | Component | Yes  | 1 | - |
+
+To see the list of all character count standards, go to this <a href="https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/wiki/Character-count-standards" target="_blank">link</a>.
+
+
 ![Feature card block medium](../../images/pattern/feature-card-block/feature-card-block-medium.jpg)
 
 ## Feature card block &mdash; large
 Feature card block &mdash; large is used to present main piece of content as a main feature.
 
+### Content guidance
+
+| Field | Field type | Required   | Cardinality | Max character size (English / translated) |
+| -------- | --------- | ---------- | ---------- | ----------|
+| Eyebrow | Short copy | Yes   | 1 | Mini (25 / 35)|
+| Headline | Short copy | Yes   | 1 | XS (65 / 85)|
+| Paragraph | Long copy | No   | 1 | LL (200 / 260)|
+| Image | Media | Yes   | 1 | - |
+| URL | Link | Yes   | 1 | - |
```

Comment:
Should this be a card or feature card? I see we're calling specifics out as required which might accurate. I wonder if there is a way to call out what's only different. Unless it's all different?

Reply:
@photodow Hmm yeah on this one I think most of them are different which is why I didn't call it out. I can call out below the table that this references "card" if you think that would help?